src/styles/setings/star.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+
+  &__stars {
+    display: flex;
+    gap: 4px;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__stars &__star:nth-child(1n) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__stars &__star:nth-child(1n + 5) {
+    background-image: url(../images/star.svg);
+  }
+
+  &__rev {
+    display: flex;
+    align-items: end;
```

Comment:
The value 'end' for 'align-items' is not valid. You might want to use 'flex-end' instead to align items to the end of the flex container.